juju/juju
# Unit agent worker loggers

As continuing preparatory work for merging unit and machine agents, the package level loggers need to be removed from the uniter workers.

This branch removes a bunch more, but there are still four left in subpackages of worker/uniter as of this proposal.

Still to go:
* worker/uniter/runner
* worker/uniter/runner/context
* worker/uniter/runner/jujuc
* worker/uniter/storage

## QA steps

The unit tests should still pass.
```sh
* bootstrap and deploy something
* ensure that the unit agent logs don't show workers restarting due to missing loggers
```

## Documentation changes

Internal only.
